Automattic/wp-calypso
# Render RSS Feed consistently between Writing settings screen and Podcasting Details screen

- RSS Feed label was missing from Podcasting Details screen
- Learn more link was missing from Podcasting Details screen
- Font style and color of informational text under RSS feed differed between Writing settings screen and Podcasting Details screen

Before:

<img width="734" alt="screencapture at thu jul 5 15 25 53 edt 2018" src="https://user-images.githubusercontent.com/2098816/42343677-d6c00cd0-8067-11e8-801f-38f4c55b504e.png">

<img width="735" alt="screencapture at thu jul 5 15 26 00 edt 2018" src="https://user-images.githubusercontent.com/2098816/42343687-dc8568e0-8067-11e8-94eb-017962a82576.png">

After:

<img width="734" alt="screencapture at thu jul 5 15 23 30 edt 2018" src="https://user-images.githubusercontent.com/2098816/42343691-e1d750ce-8067-11e8-8fe3-cbd185df1cbe.png">

<img width="738" alt="screencapture at thu jul 5 15 23 40 edt 2018" src="https://user-images.githubusercontent.com/2098816/42343698-e5f685e4-8067-11e8-907b-9d250115f21c.png">

To test:

With feature flag enabled (such as in development):

- View Writing settings screen and Podcasting Details screen and verify that RSS Feed and associated text is rendered the same on both screens
